cmd/zetatool/config/config.go

```diff
@@ -0,0 +1,81 @@
+package config
+
+import (
+	"encoding/json"
+
+	"github.com/spf13/afero"
+)
+
+var AppFs = afero.NewOsFs()
+
+const (
+	Flag                      = "config"
+	defaultCfgFileName        = "InboundTxFilter_config.json"
+	ZetaURL                   = "http://46.4.15.110:1317" //http://100.71.167.102:26657
+	TssAddressBTC             = "bc1qm24wp577nk8aacckv8np465z3dvmu7ry45el6y"
+	TssAddressEVM             = "0x70e967acfcc17c3941e87562161406d41676fd83"
```

Comment:
This what can be fetched dynamically from the chain API, I don't think we need to have those in the config